lerobot/common/policies/normalize.py

```diff
@@ -79,28 +79,48 @@ def create_stats_buffers(
             )
 
         # TODO(aliberts, rcadene): harmonize this to only use one framework (np or torch)
-        if stats:
-            if isinstance(stats[key]["mean"], np.ndarray):
-                if norm_mode is NormalizationMode.MEAN_STD:
+        if stats and key in stats:
+            # NOTE:(maractingi, azouitine): Change the order of these conditions because in online environments we don't have dataset stats
+            # Therefore, we don't access to full stats of the data, some elements either have min-max or mean-std only
```

Comment:
We will remove the comment from the code.
Change the order of these conditions because in online environments we don't have dataset stats. Therefore, we don't access to full stats of the data, some elements either have min-max or mean-std only